documentation/GETTING_STARTED.md

```diff
@@ -0,0 +1,349 @@
+# Getting Started: how to process data for Timelapse Feature Explorer
+
+The [Timelapse Feature Explorer (TFE)](https://timelapse.allencell.org) is a web-based application designed for the interactive visualization and analysis of segmented time-series microscopy data! Data needs to be processed into a specific format to be loaded into the viewer.
+
+In this tutorial, you'll learn how to prepare your data for the Timelapse Feature Explorer.
+
+## 1. Terms
+
+A few key terms:
+
+- **Dataset**: A dataset is a single time-series, and can have any number of tracked objects and features.
+- **Raw dataset**: The raw data that you have collected or generated, before processing into the TFE format.
+- **Collection**: An arbitrary grouping of datasets.
+- **Object ID**: An ID associated with a single segmentation at a single timepoint. In the TFE-accepted format, object IDs must be sequential, starting from 0, and be unique across the whole dataset.
+- **Track ID**: An identifier for a unique set of objects, linking their object IDs across timepoints.
+
+## 2. Prerequisites
+
+### Installing `colorizer-data`
+
+From a command terminal, clone this repository and run the following commands to install dependencies. This will install the necessary libraries for the example scripts and the latest release of `colorizer-data`. (You may want to do this from a virtual Python environment-- see [venv](https://docs.python.org/3/library/venv.html) or [conda](https://docs.conda.io/en/latest/) for more information.)
```

Comment:
Do you think we should have the install instructions use a virtual environment? I know in my early python days, I wasn't fluent with virtual environments and I often got myself into wild error states from conflicting dependency, and at that phase I was also likely to directly copy / paste provided commands into my terminal without following other text recommendations (which may be more of a me problem lol 😭 )

Reply:
I can reference some of the environment tutorial code that Philip wrote for NucMorph!

Reply:
Added `venv` instructions!